source/codegen/CGBuilder.h

```diff
@@ -75,7 +75,9 @@ class CGBuilder : public llvm::IRBuilder<llvm::ConstantFolder, llvm::IRBuilderDe
         llvm::TypeSize size = dl.getTypeAllocSize(elementType->getElementType());
         llvm::Align align(llvm::MinAlign(addr.getAlignment().value(), size * index));
 
-        return Address(CreateInBoundsGEP(addr.getPointer(), { getSize(0), getSize(index) }), align);
+        llvm::Value* Ptr = addr.getPointer();
```

Comment:
Can you change these to follow the naming convention of the project? For local variables, camelCase is used.